src/gs-utils.c

```diff
@@ -357,4 +357,60 @@ gs_utils_unlink (const gchar *filename, GError **error)
 	return TRUE;
 }
 
+static gboolean
+gs_utils_rmtree_real (const gchar *directory, GError **error)
```

Comment:
This is just so it prints only one message about removing the directory (instead of several if we put the debug in the recursively called function).